app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb

```diff
@@ -12,14 +12,33 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Credential < Mana
 
   FRIENDLY_NAME = "Ansible Automation Inside Credential".freeze
 
-  def self.provider_params(params)
-    super.merge(:organization => ManageIQ::Providers::EmbeddedAnsible::AutomationManager.first.provider.default_organization)
+  include ManageIQ::Providers::EmbeddedAnsible::CrudCommon
+
+  def self.params_to_attributes(_params)
+    raise NotImplementedError, "must be implemented in a subclass"
   end
 
   def self.notify_on_provider_interaction?
     true
   end
 
+  def self.raw_create_in_provider(_manager, params)
+    create!(params_to_attributes(params))
+  end
+
+  def self.notification_excludes
+    [:password, :auth_key, :service_account]
```

Comment:
Are we using service account anywhere? I thought we decided against that.

Reply:
These are just the 3 `encrypt_column` values found in `app/models/authentications`, and have nothing to do with what we are using in the sub models.  I just used this is a generic set of values to exclude based on the top level model that should never be included in the notifications.

Granted, this might not even matter if we are just dropping notifications for this to begin with.